### ISSUE TITLE
uploading photo ..............

### ISSUE THREAD

**Issue by rcrath** [theriex/membic#29]
Manjaro linux
Chrome, up to date.

replicate:
1. login, go to profile
2. upload 53K jpg file for profile
3. wait.....it says uploading, but never uploads though.

**Comment by theriex** [comment]
The problem was in a function that had been added to support the relatively new mail-in membic feature.  Specifically pen.py reflect_mailin_address_changes was unpacking PenName.stash (the quick access reference area for the PenName) but failing to deal with the situation if nothing had been stashed yet.  Tested by creating a new account and uploading an image.  Should be all set now.  Let me know if you run into any further issues.  Thanks for reporting it!

**Comment by rcrath** [comment]
Cool. It's like I know the dev or something.

**Comment by rcrath** [comment]
@theriex still getting the eternal "Uploading . . . " on a 1.1 m file as well as a 53 kb version.  tested on both an ipad in safari and chrome, and on linux box with chrome.

**Comment by theriex** [comment]
Pic upload was broken due to a bad form submit call. Firefox continued after the error, which caused the form to submit and the update to go through.  Webkit (chrome/safari) flagged the error and stopped, leading to the eternal wait.  The offending call has been fixed.  I've uploaded a new profile pic using Mac Chrome to verify things are now working.  Let me know if you run into anything else. Thanks!
